sst/SimEngMemInterface.cc

```diff
@@ -216,6 +233,24 @@ void SimEngMemInterface::aggregatedReadResponses(
   }
   // Send the completed read request back to SimEng via the
   // completed_read_requests queue.
+  uint64_t resp = 0;
+  for (int x = mergedData.size() - 1; x >= 0; x--) {
+    resp = (resp << 8) | mergedData[x];
+  }
+  // SST output data parsed by the testing framework.
+  // Format:
+  // [SimEng:SSTDebug] MemRead-read-<type=request|response>-<request ID>
+  // -cycle-<cycle count>-data-<value>
+  uint64_t id = aggrReq->id_;
+#ifdef SIMENG_ENABLE_SST_TESTS
+  id = aggrReq->testId;
```

Comment:
Why do we need to change the id?

Reply:
This change was done yesterday while I was exploring inorderpipeline archtypes, wherein multiple requests didn't have unique ids hence I was forced was resort to this. I will revert this change now.

Reply:
Please check